tools/serverpod_cli/test/generator/dart/client_code_generator/enum_field_test.dart

```diff
@@ -58,6 +59,36 @@ void main() {
     });
   });
 
+  group('Given enum named Example when generating code', () {
```

Comment:
Suggestion: Include how it is serialized in name.

To have full context when it fails in our pipeline.